tests/extensions/tabular/test_renderer.py

```diff
@@ -117,3 +136,30 @@ def test_render_tabular_xlsx_invalid(self, invalid_xlsx_file_path, assets_url, e
         renderer = TabularRenderer(metadata, invalid_xlsx_file_path, url, assets_url, export_url)
         with pytest.raises(XLRDError):
             renderer.render()
+
+
+class TestTabularMatRenderer:
+
+    def test_render_tabular_mat70(self, test_mat70_file_path, assets_url, export_url):
+        metadata = ProviderMetadata('test', '.mat', 'text/plain', '1234', 'http://wb.osf.io/file/testVer70.mat?token=1234')
+        renderer = TabularRenderer(metadata, test_mat70_file_path, url, assets_url, export_url)
+        body = renderer.render()
+        assert '<div id="mfrGrid" style="min-height: {}px;">\n    </div>'.format(settings.TABLE_HEIGHT) in body
```

Comment:
line length, and if this is a common testing var (it's used again in `test_render_tabular_mat73`) maybe make it a fixture or global.